www/js/app.js

```diff
@@ -496,6 +496,25 @@ document.getElementById('disableDragAndDropCheck').addEventListener('change', fu
         }
     });
 });
+// Handle switching from jQuery to serviceWorker modes.
+document.getElementById('serviceworkerModeRadio').addEventListener('click', function() {
+    if (this.checked) {
+        document.getElementById('enableSourceVerificationCheckBox').style.display = 'block' 
```

Comment:
Better to use `style.display = '';` rather than guessing if it's `block` or `inline`.

Also note missing semicolon at end of line.